plugins/color_panel/.gitignore

```diff
@@ -1,3 +1,8 @@
 .packages
 .flutter-plugins
 pubspec.lock
+.flutter-plugins-dependencies
+package_config.json
+flutter_export_environment.sh
+Flutter-Generated.xcconfig
+GeneratedPluginRegistrant.swift
```

Comment:
Several of these are macOS-specific, and should go in the macos directory.

Reply:
reverted